.travis.yml

```diff
@@ -76,10 +73,11 @@ before_install:
 
 install:
   - if [[ "$INSTALL_DEPS" == 1 ]]; then pip install numpy==$NUMPY_VERSION coverage pycodestyle; fi
+  - pip install git+https://github.com/lcogt/banzai@0.20.1-40-g7902cb9
```

Comment:
This pip install is a hacky way to have travis install the relevant branch of banzai: fix/nres_dbs. Adding this into the dependency links of setup.py did not cooperate, but I would rather do that since it is a lot cleaner.